README.md

```diff
@@ -128,6 +143,7 @@ plugins: [{ src: "~/plugins/vue-paycard.js", mode: "client" }];
 | is-card-number-masked  | Boolean          | true                                                                                                                                | Hides the numbers provided and changes to "\*". Only shows the last four digits                                                                                                                                                                                                                                                                                                      |
 | has-random-backgrounds | Boolean          | true                                                                                                                                | Set a random background image to the card. You can check all the images in `src/assets/images`                                                                                                                                                                                                                                                                                       |
 | background-image       | [String, Number] | ''                                                                                                                                  | Set a background image link to the card (overrides `has-random-backgrounds` prop), or you can pass a single valid number that matches the images name we have in `src/assets/images`                                                                                                                                                                                                 |
+| set-type               | [String]         | ''                                                                                                                                  | Set a card type from the supported card types below, overrides the one generated from card number                                                                                                                                                                                                                                                                                    |
```

Comment:
Please remove the brackets here for String, they are unnecessary, and if you don't mind, add this to the available types:  visaelectron, visa, elo, amex, mastercard, discover, unionpay, troy, dinersclub, jcb, laser, dankort, uatp, mir, hipercard, aura, maestro.

This way everyone will know which type they can pass in to, to force that.